docker/metrics/Makefile

```
@@ -0,0 +1,78 @@
+#!/usr/bin/make
+
+ifneq (,$(wildcard ./.env))
+    -include .env
+    -include .envrc
+    export
+endif
+
+NAME = "metrics"
+ENV_FILE ?= .env
+MOUNT_POINT ?= /mnt
+DOCKER_COMPOSE=docker-compose --env-file=${ENV_FILE}
+
+.DEFAULT_GOAL := up
+# avoid target corresponding to file names, to depends on them
+.PHONY: *
+
+#------#
+# Info #
+#------#
+info:
+	@echo "${NAME} version: ${VERSION}"
+
+goodbye:
+	@echo "🥫 Cleaning up dev environment (remove containers, remove local folder binds, prune Docker system) …"
+
+#----------------#
+# Docker Compose #
+#----------------#
+up:
+# creates a docker network and runs docker-compose
+	@echo "🥫 Building and starting containers …"
+ifdef service
+	${DOCKER_COMPOSE} up -d ${service} 2>&1
+else
+	${DOCKER_COMPOSE} up -d 2>&1
+endif
+
+down:
+	@echo "🥫 Bringing down containers …"
+	${DOCKER_COMPOSE} down
+
+hdown:
+	@echo "🥫 Bringing down containers and associated volumes …"
+	${DOCKER_COMPOSE} down -v
+
+restart:
+	@echo "🥫 Restarting containers …"
+	${DOCKER_COMPOSE} restart
+
+status:
+	@echo "🥫 Getting container status …"
+	${DOCKER_COMPOSE} ps
+
+log:
+	@echo "🥫 Reading logs (docker-compose) …"
+	${DOCKER_COMPOSE} logs -f
+
+#------------#
+# Production #
+#------------#
+create_external_volumes:
+	@echo "🥫 Creating external volumes (production only) …"
+	docker volume create grafana-storage
+	docker volume create influx-storage
```

If you change volume names as propoesd in prod.yml:

```suggestion
	docker volume create ${COMPOSE_PROJECT_NAME}_grafana-storage
	docker volume create ${COMPOSE_PROJECT_NAME}_influx-storage
```